src/theme/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { createTheme, Color } from '@mui/material'
+declare module '@mui/material/Button' {
+  interface ButtonPropsVariantOverrides {
+    template: true
+    createTemplate: true
+  }
+}
+
+let theme = createTheme({
+  palette: {
+    primary: {
+      main: '#2da6ca',
+    },
+  },
+  components: {
+    MuiButton: {
+      variants: [
```

Comment:
Shouldn't be creating variants(template, createTemplate) in Theme File.
Need to create using (sx or styles or styled Component) in pages.

Reply:
Removed custom varinat